### PR TITLE
Layers order parameter

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?

* Ticket: #
* Funded by: DFAT
* Description: 
  * Headless needs a call for generating report where the client can give a parameter for layer order in the map report.

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/report/test/test_impact_report.py [hunk 1]
```diff
@@ -2192,6 +2207,90 @@ def test_qgis_map_pdf_report(self):
...
 
+    def test_custom_layer_order_qgis_map_pdf_report(self):
+        """Test generate map report using custom layer order.
+
+        .. versionadded: 4.3.2
+        """
+
+        """Single Exposure ImpactFunction"""
+
+        output_folder = self.fixtures_dir('../output/impact_map_pdf')
```
Comment (ismailsunni) [inline safe/report/test/test_impact_report.py]: Avoid to use `/`, it may not work in Windows.

**Comment by Gustry** [comment]
Is it a feature or a bug fix?

This PR is targeting develop. Just thinking that it's still 4.3 in develop.

**Comment by ismailsunni** [comment]
@myarjunar can you merge this? Is it ready now?

**Comment by myarjunar** [comment]
yes it is already green. thanks @ismailsunni

**Pull request merged**